GOVERNANCE.md

```diff
@@ -0,0 +1,201 @@
+# ESLint-Community Governance
+
+## Core Team
```

Comment:
The RFC & [announcement blogpost](https://eslint.org/blog/2023/03/announcing-eslint-community-org) are talking about `Community Core team`, so maybe we should keep doing that here as well?

```suggestion
## Community Core team
```

Reply:
Since this is a document for the `ESLint-Community` organization I used `Core Team` rather than `Community Core team`. The latter would make sense if this was an `ESLint` document, but it isn't

Reply:
We used the term "Community Core Team" in the post to disambiguate between the ESLint core team, which was important in that context.

In this context, I think it's fine to shorten to "Core Team", as it's inside of the organization so there is no ambiguity.